roachtest: activerecord failed

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2582336&tab=buildLog) on [release-20.2@c9a9087632ea8616b5e7c1e897012c12e0b6e80c](https://github.com/cockroachdb/cockroach/commits/c9a9087632ea8616b5e7c1e897012c12e0b6e80c):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-140-gc9a9087632
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2582336&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2589104&tab=buildLog) on [release-20.2@f49d66a2c3a7696c3ffa4c873f45673d9a640cf6](https://github.com/cockroachdb/cockroach/commits/f49d66a2c3a7696c3ffa4c873f45673d9a640cf6):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.3-151-gf49d66a2c3-dirty
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6092 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2589104&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2601092&tab=buildLog) on [release-20.2@d18660ddabd1929bc74a50492491ec0fdfa1a41b](https://github.com/cockroachdb/cockroach/commits/d18660ddabd1929bc74a50492491ec0fdfa1a41b):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-35-gd18660ddab
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6090 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyTest#test_schema_dumping_with_custom_fk_ignore_pattern (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2601092&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2618903&tab=buildLog) on [release-20.2@eb21aede5ceae7747d1392b66b61ec7799d5171c](https://github.com/cockroachdb/cockroach/commits/eb21aede5ceae7747d1392b66b61ec7799d5171c):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-55-geb21aede5c
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6092 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2618903&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2629421&tab=buildLog) on [release-20.2@2ed4134be037a4b8c169518a05b2d5347602a2af](https://github.com/cockroachdb/cockroach/commits/2ed4134be037a4b8c169518a05b2d5347602a2af):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-67-g2ed4134be0
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6092 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2629421&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2635631&tab=buildLog) on [release-20.2@526edb4cfbc67f3c4c3468b48fa51fd8c73776b6](https://github.com/cockroachdb/cockroach/commits/526edb4cfbc67f3c4c3468b48fa51fd8c73776b6):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-70-g526edb4cfb
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6092 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2635631&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2646713&tab=buildLog) on [release-20.2@162c5ac4968cf31c0ed54cd29aa8aeccd66247bb](https://github.com/cockroachdb/cockroach/commits/162c5ac4968cf31c0ed54cd29aa8aeccd66247bb):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-76-g162c5ac496-dirty
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2646713&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657161&tab=buildLog) on [release-20.2@8c79e2bc4b35d36c8527f4c40c974f03d9034f46](https://github.com/cockroachdb/cockroach/commits/8c79e2bc4b35d36c8527f4c40c974f03d9034f46):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-78-g8c79e2bc4b
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6090 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::Migration::ForeignKeyTest#test_schema_dumping_with_custom_fk_ignore_pattern (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657161&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2665956&tab=buildLog) on [release-20.2@bd46ccc517326f2a2d7be69defc8ca2f91172b1b](https://github.com/cockroachdb/cockroach/commits/bd46ccc517326f2a2d7be69defc8ca2f91172b1b):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-85-gbd46ccc517
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2665956&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2673431&tab=buildLog) on [release-20.2@243d8efaccee1f21833718b237fbaf9ca3d4a88d](https://github.com/cockroachdb/cockroach/commits/243d8efaccee1f21833718b237fbaf9ca3d4a88d):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.4-89-g243d8efacc-dirty
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2673431&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688450&tab=buildLog) on [release-20.2@6b701b545f1a50c8e9b2de216b0d8d02cd7d0d72](https://github.com/cockroachdb/cockroach/commits/6b701b545f1a50c8e9b2de216b0d8d02cd7d0d72):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-23-g6b701b545f
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688450&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693255&tab=buildLog) on [release-20.2@a1b85f53c40bb620f75aec4cd4f676397cb8794b](https://github.com/cockroachdb/cockroach/commits/a1b85f53c40bb620f75aec4cd4f676397cb8794b):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-25-ga1b85f53c4
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693255&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699382&tab=buildLog) on [release-20.2@35acc871eba44f74264e4157457e15fb74323975](https://github.com/cockroachdb/cockroach/commits/35acc871eba44f74264e4157457e15fb74323975):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-31-g35acc871eb
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699382&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2707864&tab=buildLog) on [release-20.2@0932fedf3e04137494f5df7131025cca02c77d62](https://github.com/cockroachdb/cockroach/commits/0932fedf3e04137494f5df7131025cca02c77d62):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-35-g0932fedf3e
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2707864&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2712443&tab=buildLog) on [release-20.2@02ead6473c49c8f2b30c553d6ddc24f0e8a63333](https://github.com/cockroachdb/cockroach/commits/02ead6473c49c8f2b30c553d6ddc24f0e8a63333):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-43-g02ead6473c
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2712443&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716844&tab=buildLog) on [release-20.2@f2da720e3bc8da23edf3f09fe8165c9b3ced273d](https://github.com/cockroachdb/cockroach/commits/f2da720e3bc8da23edf3f09fe8165c9b3ced273d):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-50-gf2da720e3b
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716844&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721403&tab=buildLog) on [release-20.2@928adc756f5dc984feb74b863a1c94d54ed658d8](https://github.com/cockroachdb/cockroach/commits/928adc756f5dc984feb74b863a1c94d54ed658d8):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-54-g928adc756f
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6091 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		3 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721403&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2725148&tab=buildLog) on [release-20.2@c82e01111406152c655777087a5bc783dc7becf6](https://github.com/cockroachdb/cockroach/commits/c82e01111406152c655777087a5bc783dc7becf6):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-63-gc82e011114
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2725148&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726630&tab=buildLog) on [release-20.2@22e282d6cda8348ec39445106e8b10a5663297c8](https://github.com/cockroachdb/cockroach/commits/22e282d6cda8348ec39445106e8b10a5663297c8):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-65-g22e282d6cd
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726630&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733693&tab=buildLog) on [release-20.2@34ae57c277484ddc95e6f01137a75e0267c887b7](https://github.com/cockroachdb/cockroach/commits/34ae57c277484ddc95e6f01137a75e0267c887b7):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-80-g34ae57c277
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6093 tests passed
		1 test failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733693&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2738846&tab=buildLog) on [release-20.2@874752451cbec692241efc19b84f7c3fecbaf469](https://github.com/cockroachdb/cockroach/commits/874752451cbec692241efc19b84f7c3fecbaf469):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-86-g874752451c
		Tests run against activerecord 6.0.3.4
		6141 Total Tests Run
		6133 tests passed
		8 tests failed
		58 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: SpatialQueriesTest#test_query_st_distance_from_constant (unexpected)
		--- FAIL: SpatialQueriesTest#test_query_st_distance (unexpected)
		--- FAIL: SpatialQueriesTest#test_ewkt_parser_query (unexpected)
		--- FAIL: SpatialQueriesTest#test_query_rgeo_bbox_node (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: SpatialQueriesTest#test_query_st_length (unexpected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- FAIL: SpatialQueriesTest#test_query_rgeo_feature_node (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2738846&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2743180&tab=buildLog) on [release-20.2@91d34442ff435b5d0eba7124aae99fa71cecc0d7](https://github.com/cockroachdb/cockroach/commits/91d34442ff435b5d0eba7124aae99fa71cecc0d7):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:227,activerecord.go:239,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-98-g91d34442ff
		Tests run against activerecord 6.0.3.4
		6141 Total Tests Run
		6140 tests passed
		1 test failed
		58 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2743180&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2747348&tab=buildLog) on [release-20.2@ba484eddf4fbb9d4e79c6d9d744c15ffbbeb36db](https://github.com/cockroachdb/cockroach/commits/ba484eddf4fbb9d4e79c6d9d744c15ffbbeb36db):

```
The test failed on branch=release-20.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:227,activerecord.go:239,test_runner.go:755: 
		Tests run on Cockroach v20.2.5-100-gba484eddf4
		Tests run against activerecord 6.0.3.4
		6141 Total Tests Run
		6140 tests passed
		1 test failed
		58 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2747348&tab=artifacts#/activerecord)
Related:
- #58783 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/activerecord.go
+++ b/pkg/cmd/roachtest/activerecord.go
@@ -20,8 +20,8 @@ import (
 
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
-var supportedRailsVersion = "6.0.3.4"
-var adapterVersion = "v6.0.0beta3"
+var supportedRailsVersion = "6.1"
+var adapterVersion = "v6.1.0beta2"
 
 // This test runs activerecord's full test suite against a single cockroach node.
 

